work/worker.go

```diff
@@ -88,6 +89,8 @@ var (
 	snapshotAccountReadTimer = metrics.NewRegisteredTimer("miner/snapshot/account/reads", nil)
 	snapshotStorageReadTimer = metrics.NewRegisteredTimer("miner/snapshot/storage/reads", nil)
 	snapshotCommitTimer      = metrics.NewRegisteredTimer("miner/snapshot/commits", nil)
+	oldTimer                 = metrics.NewRegisteredTimer("reward/oldtimer", nil)
+	newTimer                 = metrics.NewRegisteredTimer("reward/newtimer", nil)
```

Comment:
@ian0371 Can we rename the timer metrics to `reward/distribute/default` `reward/distribute/kip82`?

Reply:
done